CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 - Delete user functionality
 
+
```

Comment:
You also need to modify the `pluginInterfaceSupported.json` file to replace "2.10" with "2.11"

Reply:
And need to modify the build.gradle to have version 1.12

Reply:
both done.